table-module/src/main/java/com/iluwatar/tablemodule/UserTableModule.java

```diff
@@ -0,0 +1,90 @@
+package com.iluwatar.tablemodule;
+
+import java.sql.Connection;
+import java.sql.PreparedStatement;
+import java.sql.ResultSet;
+import java.sql.SQLException;
+import javax.sql.DataSource;
+
+import lombok.extern.slf4j.Slf4j;
+
+
+/**
+ * This class organizes domain logic with the user table in the
+ * database. A single instance of this class contains the various
+ * procedures that will act on the data.
+ */
+@Slf4j
+public class UserTableModule {
+  private final DataSource dataSource;
+  private Connection connection = null;
+  private ResultSet resultSet = null;
+  private PreparedStatement preparedStatement = null;
+
+  /**
+   * Public constructor.
+   *
+   * @param userDataSource the data source in the database
+   */
+  public UserTableModule(final DataSource userDataSource) {
+    this.dataSource = userDataSource;
+  }
+
+
+  /**
+   * Login using username and password.
+   *
+   * @param username the username of a user
+   * @param password the password of a user
+   * @return the execution result of the method
+   * @throws SQLException if any error
+   */
+  public int login(final String username, final String password)
+          throws SQLException {
+    try {
```

Comment:
Could we use try-with-resources statement for automatic closing? Check elsewhere too.